lib/Backend/LowerMDShared.cpp

```diff
@@ -8684,6 +8684,18 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
         Assert(helperMethod == (IR::JnHelperMethod)0);
         return GenerateFastInlineBuiltInMathAbs(instr);
 
+    case Js::OpCode::InlineMathPow:
+#ifdef _M_IX86
+        if (!instr->GetSrc2()->IsFloat())
```

Comment:
Yes, GenerateFastInlineBuiltInMathPow should handle all except the customized calling convention.

---
In reply to: [57632884](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57632884) [](ancestors = 57632884)